plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/MavenPathParser.java

```diff
@@ -10,7 +10,7 @@
  * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
  * Eclipse Foundation. All other trademarks are the property of their respective owners.
  */
-package org.sonatype.nexus.repository.maven.internal;
+package org.sonatype.nexus.repository.maven;
```

Comment:
Moved it out of internal to make it visible in PaxIT w/o the "global" classloader fuss. This is really a technical issue, we can make it internal again, but then IT needs to be modified to use "über" classloader or the helper component to look up internal components.